pandas/tests/indexing/test_partial.py

```diff
@@ -637,3 +637,66 @@ def test_partial_set_empty_frame_empty_consistencies(self):
         df.loc[0, 'x'] = 1
         expected = DataFrame(dict(x=[1], y=[np.nan]))
         tm.assert_frame_equal(df, expected, check_dtype=False)
+
+    def test_access_timezoned_datetimeindex_with_timezoned_label_utc(self):
```

Comment:
I think this is the incorrect file for the tests, as the partial in `test_partial.py` is not referring partial datetime string indexing, but partial setting or something. 
There is a `indexes/datetimes/test_partial_slicing.py` that seems a better fit.

Reply:
Possible, this was a suggestion of @jreback.

Reply:
Yes, but I think it was a typo of @jreback and he forgot the _slicing part. You can look in the current file and see that there is no test related to datetime index slicing, so please make the change